### ISSUE TITLE
[App]: Can't log table - won't show up in Run Tables

### ISSUE THREAD

**Issue by michaelabuchanan** [wandb/wandb#5009]
### Current Behavior

I am trying to log a dataset from a Jupyter notebook as a Table. When I run the code it creates a Table in my Workspace but says "No rows to display" instead of showing the data I tried to log. However, I can tell that the data is passed to WandB because if I navigate to Files/media/table and download the `.json` file located there I can see the data I am trying to pass in the `.json` file. To make sure it wasn't my code causing issues I tried running this sample notebook https://colab.research.google.com/github/wandb/examples/blob/master/colabs/tables/W%26B_Tables_Quickstart.ipynb and still was not able to see the table populated for the runs I did.
![tableissues](https://user-images.githubusercontent.com/32206926/220457687-9379ec84-c844-417f-885a-3e4b16839505.PNG)

### Expected Behavior

I am expecting to see the Iris dataset from the example notebook linked above populated into the Table created by running this code.

### Steps To Reproduce

1. Run the notebook linked in Current Behavior
2. Check if the table is created as expected in the WandB web application

### Screenshots

.json file found under Files:
![jsonfile](https://user-images.githubusercontent.com/32206926/220458501-08eea77e-ee40-47a2-858c-efd1b8ede284.PNG)

Screenshot of notebook used:
![notebook](https://user-images.githubusercontent.com/32206926/220458704-078ba480-b81d-4fd8-9073-a1ea7f8f41c5.PNG)

### Environment

OS: Windows 10

Browsers: Google Chrome

Version: 0.13.10

### Additional Context

I am using Weights and Biases through Run.AI so perhaps something about this setup is causing this issue. Normal Keras training logging is working as expected. So far it's only the Tables that I have seen not work.

**Comment by MBakirWB** [comment]
Hi @michaelabuchanan, in the screenshot provided the dataframe is not being logged correctly to a wandb table. The call must be `wandb.log({"iris":  wandb.Table(dataframe=iris_dataframe})`

You can test the correct behavior via the following. table 1 will not appear, but table 2 will. 

```
run1 = wandb.init(project="test-plot-table")
ids = np.arange(10)

df1 = pd.DataFrame({"id": ids, "X": np.random.randint(0, 5, size=10)})
table1 = run1.log({"table1": df1})

df2 = pd.DataFrame({"id": ids, "Y": np.random.randint(0, 5, size=10)})
table2 = run1.log({"table2": wandb.Table(dataframe=df2)})
wandb.finish()
```

**Comment by michaelabuchanan** [comment]
Thank you for the quick reply! I tried out both changing the code as mentioned and plugging in the sample code you provided and still don't see output from any of the tables. I can once again see the data in the .json files though...

**Comment by MBakirWB** [comment]
Hi @michaelabuchanan , I accessed the latest version of your jupyter notebook and successfully logged the tables, see [here](https://wandb.ai/mohammadbakir/Tables-Quickstart/artifacts/code/source-Tables-Quickstart-None/v0/files/_session_history.ipynb). I noticed you are using a private wandb instance, could you run `wandb verify` from your terminal to check the state of your installation. Additionally, when you could, provide me a screen shot of your browser console from from the main page where you are not seeing the data. Thanks

**Comment by michaelabuchanan** [comment]
Here's screenshots of both the wandb verify output and my browser console. Looks like it's not happy about the JSON file it's getting for the table...
![terminal](https://user-images.githubusercontent.com/32206926/221921192-9f441a49-6469-436d-a22c-0e8e22a09265.PNG)
![browserconsole](https://user-images.githubusercontent.com/32206926/221921244-c93956e1-fa84-4ba4-8d49-179fd259267e.PNG)

**Comment by MBakirWB** [comment]
Hi @michaelabuchanan , since we have not heard back from you we are going to close this request. If you would like to re-open the conversation, please let us know!

**Comment by KohakuBlueleaf** [comment]
@MBakirWB I also meet this problem frequently in this 2 months.
And the same code can work normally previous.

Is there any updates for Table these 2~3 months?

**Comment by KohakuBlueleaf** [comment]
And the behavior is very weird.
I logged a table before training loop and it can show up correctly.
But after I start training loop and log the loss or something related to. The table just disapper (no row to display)

**Comment by jmj23** [comment]
I seem to be having a similar issue. I am running a training script (not a notebook) and initialize and populate the table as below:
 ```
# setup wandb config
if args.use_wandb:
    run_name = f"{args.model_version}_{args.max_num_frames}_{args.img_res}"
    wandb_run = wandb.init(
        project=f"{args.project}_transformers",
        config=args_dict,
        name=run_name,
    )
    wandb_logging_steps = global_iters_per_epoch // 10
else:
    wandb_run = None
```
later
```
if args.use_wandb:
    columns = ["image_path", "gt_caption", "pred_caption"]
    table = wandb.Table(data=wandb_data, columns=columns)
    wandb_run.log({"examples": table}, step=global_step)
```
where `wandb_data` is a list of lists, as in the documentation.

As in the OP, the table data shows up in "artifacts" under "files" in the UI, but the table inside the "runs" will not populate:
![image](https://github.com/wandb/wandb/assets/32778995/f896f331-80a0-49dc-b3af-628fd70df326)

**Comment by jayagami** [comment]
> I seem to be having a similar issue. I am running a training script (not a notebook) and initialize and populate the table as below:
> 
> ```
> # setup wandb config
> if args.use_wandb:
>    run_name = f"{args.model_version}_{args.max_num_frames}_{args.img_res}"
>    wandb_run = wandb.init(
>        project=f"{args.project}_transformers",
>        config=args_dict,
>        name=run_name,
>    )
>    wandb_logging_steps = global_iters_per_epoch // 10
> else:
>    wandb_run = None
> ```
> 
> later
> 
> ```
> if args.use_wandb:
>     columns = ["image_path", "gt_caption", "pred_caption"]
>     table = wandb.Table(data=wandb_data, columns=columns)
>     wandb_run.log({"examples": table}, step=global_step)
> ```
> 
> where `wandb_data` is a list of lists, as in the documentation.
> 
> As in the OP, the table data shows up in "artifacts" under "files" in the UI, but the table inside the "runs" will not populate: ![image](https://user-images.githubusercontent.com/32778995/239929865-f896f331-80a0-49dc-b3af-628fd70df326.png)

Same with me.

**Comment by kidkych** [comment]
Having the same problem. I noticed earlier that the table would populate after a run finishes, but while the run is active the table only contains the above mentioned message "No rows to display".

![pmpd Workspace – Weights   Biases - Google Chrome_001](https://github.com/wandb/wandb/assets/10579770/996b9175-d145-423c-a54b-3cbef46cbff8)


If I navigate to the the table under artifacts it does render correctly, however it makes it difficult to compare different runs as the bars for each key in the combined table view are the same color.

![pmpd Artifacts – Weights   Biases - Google Chrome_001](https://github.com/wandb/wandb/assets/10579770/8f2cd0dd-ea40-42b8-af47-37b4f6d76b74)

**Comment by KohakuBlueleaf** [comment]
After update to wandb 0.15.3, this issue been resolved (for me)

**Comment by jmj23** [comment]
This seems to have resolved for me as well with the update to 0.15.3

**Comment by VityaVitalich** [comment]
This issue has not resolved for me even updated to latest version

**Comment by kxhit** [comment]
Hi, I met the same issue with wandb==0.15.7. The same wandb code in my script works well for training log, but shows "No rows to display" for my val log. Thus I don't think its my code issue. Could someone help me? I'm so confused.

Everything works fine with list of images but can't upload more than 108 images, so I changed to table and put wandb image into it.

**Comment by pzpzpzp2** [comment]
I'm running into this issue frequently with version 0.15.8

**Comment by SumanthRH** [comment]
Just putting this out here for someone else trying to figure this out: I've had the same issue with version 0.15.12. Simply downgrading to 0.15.3 worked for me! Need to investigate to see why this happened though. I've followed everything in the official [guide](https://docs.wandb.ai/guides/tables/visualize-tables)

**Comment by pzpzpzp2** [comment]
The docs also seem only to say tables need to be logged via run.log, while everything else can be logged with wandb.log. I don't get why but could that have to do with this bug?

**Comment by isdanni** [comment]
Following, wandb version is 0.16.0

**Comment by sydholl** [comment]
We are closing this issue due to inactivity. If you feel that it merits additional review, please provide a reproduction and add a comment explaining why. Thank you.

**Comment by skengman1312** [comment]
I just upgraded to wandb 0.19.0 and I am still faceing the same issue

**Comment by rerz** [comment]
I am having the same issue on wandb 0.19.1.

**Comment by TobiasBrx** [comment]
having the same issue with wandb 0.18.6 and wandb 0.19.6

**Comment by Neismo** [comment]
I am also having this issue as of 0.19.6 versioning

**Comment by njwfish** [comment]
I am also having this issue on wandb 0.19.10. Does anyone know of a fix?

**Comment by BBerabi** [comment]
@sydholl This is still an issue on the latest version, 0.20.1. Can we re-open this issue?

Some of the tables are not displayed correctly, the title is missing and some of the tables are not displayed at all, although the data is available under files/artifacts.

**Comment by thewh1teagle** [comment]
Having the same issue

```python
    table_data = []
    for idx in random_indices:
        table_data.append([all_ground_truth[idx], all_predictions[idx]])

    # Debug: Check if we have data
    print(f"🔍 Debug: Creating wandb table with {len(table_data)} rows")
    for i, row in enumerate(table_data):
        print(f"   Row {i}: Source='{row[0][:50]}...', Prediction='{row[1][:50]}...'")

    examples_table = wandb.Table(columns=["Source", "Prediction"], data=table_data)
    wandb.log({"Examples_Table": examples_table}, step=step)
```

**Comment by WitchPuff** [comment]
same issue on wandb 0.21.0. I am able to find the corresponding json file in file/media/table, but fail to visualize it as a table

```python
table = wandb.Table(columns=["y", "prediction"])

for record in predictions:
    y = record["y"]
    pred = record["prediction"]
    table.add_data(str(y), str(pred))

run.log({f"{part}-predictions": table}, step=epoch)
print(f"Logged {part} predictions to wandb.")
```

**Comment by toilaluan** [comment]
Same here, looks like wb's lifetime issue

**Comment by DanielHalachev** [comment]
I experience the same issue, the very same code used to work 7 months earlier.
